src/backend/services/users-api/src/app/routes.py

```diff
@@ -0,0 +1,114 @@
+from src.app import app
+from flask import request, jsonify
+from src.app import mongo
+import sys, os
+
+if os.environ.get('SERVICE_IN_DOCKER', False):
+    sys.path.append(os.path.abspath(os.path.join('..', '')))
+else:
+    sys.path.append(os.path.abspath(os.path.join('../..', '')))
+
+from shared import HttpCode, FailureReturnString, config
+
+
+@app.route("/")
+def index():
+    return jsonify(
+        status=True,
+        message="Hello Users of " + mongo.db.name
+    )
+
+
+@app.route("/login", methods=["POST"])
+def login():
+    data = request.get_json(force=True)
+
+    try:
+        username = data['username']
+        password = data['password']
+    except Exception as e:
+        return jsonify(err= FailureReturnString.INCORRECT_PAYLOAD.value), HttpCode.BAD_REQUEST.value
+
+    user = mongo.db.users.find_one({"username":username})
+
+    return jsonify(success = (user["password"] == password))
+
+@app.route("/getUserInfo", methods=["POST"])
+def getUserInfo():
+    data = request.get_json(force=True)
+
+    try:
+        username = data['username']
+    
+    except Exception as e:
+          return jsonify(err=FailureReturnString.INCORRECT_PAYLOAD.value), HttpCode.BAD_REQUEST.value
+
+    user = mongo.db.users.find_one({'username' : username})
+
+    try:
+        data = {
+            'first_name' : user['first_name'],
+            'last_name' : user['last_name'],
+            'username' : user['username'],
+            'public_key' : user['public_key']
+        }
+    except Exception as e:
+        data = "user not found!"
+
+    return jsonify(
+        status=True,
+        data = data
+    )
+
+@app.route("/users")
+def users():
+    userList = mongo.db.users.find()
+    
+    user = {}
+    data = []
+
+    for usr in userList:
+        user = {
+            'first_name' : usr['first_name'],
+            'last_name' : usr['last_name'],
+            'username' : usr['username'],
+            'public_key' : usr['public_key']
+        }
+        data.append(user)
+
+    return jsonify(
+        status=True,
+        data=data
+    )
+
+@app.route('/users', methods=['POST'])
+def createUser():
+    data = request.get_json(force=True)
```

Comment:
I don't think you need the `force=True` parameter for POST endpoints. We used this for the GET endpoints, where a JSON is expected, unlike the POST! Should be able to remove it here and in other POST endpoints.

Reply:
Agreed

Reply:
removed